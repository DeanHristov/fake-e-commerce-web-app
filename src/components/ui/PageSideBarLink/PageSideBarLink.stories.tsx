import type { Meta, StoryObj } from '@storybook/react';
import PageSideBarLink, { IPageSideBarLinkProps } from './index';

const meta: Meta<IPageSideBarLinkProps> = {
  title: 'UI/PageSideBarLink',
  component: PageSideBarLink,
  parameters: {
    layout: 'centered',
  },
  argTypes: {},
};

export default meta;

type Story = StoryObj<IPageSideBarLinkProps>;

export const Primary: Story = {
  render: () => <PageSideBarLink />,
};
