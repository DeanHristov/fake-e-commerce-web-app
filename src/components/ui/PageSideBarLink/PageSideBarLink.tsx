import { FC } from 'react';

export interface IPageSideBarLinkProps {}

const PageSideBarLink: FC<IPageSideBarLinkProps> = ({}) => {
  return <div />;
};

export default PageSideBarLink;
