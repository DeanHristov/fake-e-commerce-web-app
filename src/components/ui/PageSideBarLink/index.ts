import PageSideBarLink, { IPageSideBarLinkProps } from './PageSideBarLink';

export type { IPageSideBarLinkProps };

export default PageSideBarLink;
