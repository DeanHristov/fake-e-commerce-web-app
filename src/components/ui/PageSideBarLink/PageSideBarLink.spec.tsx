import { render } from '@testing-library/react';
import PageSideBarLink from './index';

describe('UI/PageSideBarLink <PageSideBarLink {...}>', () => {
  it('Should render default state', () => {
    const { container } = render(<PageSideBarLink />);

    expect(container).toMatchSnapshot();
  });
});
