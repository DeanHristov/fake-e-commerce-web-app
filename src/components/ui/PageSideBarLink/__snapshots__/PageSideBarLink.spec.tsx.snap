// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/PageSideBarLink <PageSideBarLink {...}> Should render default state 1`] = `
<div>
  <div />
</div>
`;
